message.go

```diff
@@ -31,22 +31,25 @@ func (t MessageType) String() string {
 		return "Delete"
 	case MessageTypeTruncate:
 		return "Truncate"
+	case MessageTypeLogicalDecodingMessage:
+		return "Message"
 	default:
 		return "Unknown"
 	}
 }
 
 // List of types of logical replication messages.
 const (
-	MessageTypeBegin    MessageType = 'B'
-	MessageTypeCommit   MessageType = 'C'
-	MessageTypeOrigin   MessageType = 'O'
-	MessageTypeRelation MessageType = 'R'
-	MessageTypeType     MessageType = 'Y'
-	MessageTypeInsert   MessageType = 'I'
-	MessageTypeUpdate   MessageType = 'U'
-	MessageTypeDelete   MessageType = 'D'
-	MessageTypeTruncate MessageType = 'T'
+	MessageTypeBegin                  MessageType = 'B'
+	MessageTypeCommit                 MessageType = 'C'
+	MessageTypeOrigin                 MessageType = 'O'
+	MessageTypeRelation               MessageType = 'R'
+	MessageTypeType                   MessageType = 'Y'
+	MessageTypeInsert                 MessageType = 'I'
+	MessageTypeUpdate                 MessageType = 'U'
+	MessageTypeDelete                 MessageType = 'D'
+	MessageTypeTruncate               MessageType = 'T'
+	MessageTypeLogicalDecodingMessage MessageType = 'M'
```

Comment:
I suggest the type name just as `MessageTypeMessage` corresponding to protocol spec: https://www.postgresql.org/docs/current/protocol-logicalrep-message-formats.html

I also suggest put 'M' between 'B' and 'C', same as the order display in the spec.

Reply:
I agree on both suggestions.

Reply:
Given that the author has not responded for a while, do you mind if I go ahead and create a new PR with those suggestions?

Reply:
@masterkidan  I just received the notification of your new pr. I'm sorry I missed last one. You can use github cli to pull @catkins pr then add you commit, and open the new pr with this branch. 

The command like: `gh pr checkout 52`

Reply:
That doesn't really work as I don't have permissions to push to catkins repo, I get an auth failed error: 
```
fatal: Authentication failed for 'https://github.com/catkins/pglogrepl.git/'
```

Given that I've already done the work in the other PR, I'd appreciate it if you can review that.

Reply:
What I meant was that you can create a new pull request with both Catkin's and your commits. This pull request should be sent from your forked repository and contain both commits. You can do this by first checking out commit #52, and then cherry-picking this commit to your branch. After that, create a new commit with your revision.

I have reviewed your pull request #53, it’s find, but I suggest we keep Catkin's commit in the history because it is his contribution.